.github/workflows/docker-build-test.yaml

```diff
@@ -171,43 +171,8 @@ jobs:
       FEATURES: consensus-only-perf-test
       BUILD_ADDL_TESTING_IMAGES: true
 
-  rust-images-all:
```

Comment:
why there are changes to this file?

Reply:
This is from a force push by Stelian. Check Slack more context 😄